inject.js

```diff
@@ -211,10 +221,11 @@
 
     tc.videoController.prototype.initializeControls = function() {
       var document = this.document;
-      var speed = parseFloat(tc.settings.speeds[this.video.currentSrc]).toFixed(2),
```

Comment:
The intention of the original author was confusing to me, so I broke this out into their own declarations